### ISSUE TITLE
UICollectionView selectedItemIndexPath inaccessible issue

### ISSUE THREAD

**Issue by madiguzel** [DeclarativeHub/Bond#614]
Hi,

After updating to version 7.4.0 and Xcode 10.2 (Swift 5), I'm getting the error below:

**'selectedItemIndexPath' is inaccessible due to 'internal' protection level**

And the below is preview of the library code.

```
extension ReactiveExtensions where Self.Base : UICollectionView {

    /// A `ProtocolProxy` for the collection view delegate.
    ///
    /// - Note: Accessing this property for the first time will replace collection view's current delegate
    /// with a protocol proxy object (an object that is stored in this property).
    /// Current delegate will be used as `forwardTo` delegate of protocol proxy.
    public var delegate: Bond.ProtocolProxy { get }

    /// A `ProtocolProxy` for the collection view data source.
    ///
    /// - Note: Accessing this property for the first time will replace collection view's current data source
    /// with a protocol proxy object (an object that is stored in this property).
    /// Current data source will be used as `forwardTo` data source of protocol proxy.
    public var dataSource: Bond.ProtocolProxy { get }
}
```

**Comment by madiguzel** [comment]
I've just created a PR for the issue.

**Comment by srdanrasic** [comment]
Thanks for the PR, I've released it as 7.4.1.
